doc/specs/#166 - Font Support.md

```diff
@@ -0,0 +1,387 @@
+---
+author: Ryan Fu @ryfu-msft
+created on: 2024-10-1
+last updated: 2024-10-1
+issue id: 166
+---
+
+# Support for installation of fonts
+
+For [#66](https://github.com/microsoft/winget-cli/issues/166)
+
+## Abstract
+This spec outlines the design for supporting the installation of fonts.
+
+## Context
+
+### Font File Types
+
+This feature will support font files with the following file types:
+- .ttf (True Type File) - Most common format
+- .ttc (True Type Collection)
+- .otf (Open Type Font)
+
+The following font types to my knowledge are not supported:
+- .woff & .woff2 (developed by Google for the modern browser)
+- .eot (Embedded OpenType font file) Not used because of security issues.
+
+### Installing Fonts
+Typically, a user would download the font file and drag it to the `C:\Windows\Fonts` directory in the explorer view, but there are a couple things happening behind the scenes.
+
+Depending on the scope of the user, the font will be stored in two locations:
+- `C:\Windows\Fonts` (Machine )
+- `%LOCALAPPDATA%\Microsoft\Windows\Fonts` (User Mode)
+
+In addition, a new registry entry is created in the following registry paths:
+The name of the registry key typically matches the title of the font file, and the value is set to the path of the font file.
+
+#### Local Machine Registry
+---
+
+`Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Fonts`
+
+| Name    | Type | Value |
+| -------- | ------- | ------- |
+| Calibri Bold Italic (TrueType)  | REG_SZ    | calibriz.ttf |
+
+> Note that for the LOCAL_MACHINE registry entry, only the file name is specified, not the full path.
+
+#### Current User Registry
+---
+
+ `Computer\HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Fonts`
+
+| Name    | Type | Value |
+| -------- | ------- | ------- |
+| Roboto Bold Italic (TrueType)  | REG_SZ    | C:\Users\ryfu\AppData\Local\Microsoft\Windows\Fonts\Roboto-BoldItalic.ttf |
+
+
+### Uninstalling Fonts
+To remove a font, the inverse operation of the steps above need to be completed. 
+1. Remove the registry key entry for that specific font file
+2. Remove the file from the font directory
+
+
+## Manifest Changes
+- Addition of `font` to `installerType` and `nestedInstallerType`
+
+Manifest Example:
```

Comment:
Is this example missing `NestedInstallerType: font`?

Reply:
Forgot to add the one thing that I needed 😅